drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -376,6 +381,7 @@ function DragAndDropTemplates(configuration) {
             h("section.action-toolbar-item.sidebar-buttons", {}, [
                 sidebarButtonTemplate("keyboard-help-button", "fa-question", gettext('Keyboard Help')),
                 sidebarButtonTemplate("reset-button", "fa-refresh", gettext('Reset'), ctx.disable_reset_button),
+                (ctx.show_show_answer ? sidebarButtonTemplate("show-answer-button", "fa-warning", gettext('Show Answer'), ctx.disable_show_answer_button, ctx.show_answer_spinner) : null),
```

Comment:
@arbrandes This line is quite long and a little hard to read. I'd probably spell the logic out before the call to `h`:

```js
var showAnswerButton = null;
if (ctx.show_show_answer) {
    showAnswerButton = sidebarButtonTemplate(
        "show-answer-button",
        "fa-warning",
        gettext('Show Answer'),
        ctx.disable_show_answer_button,
        ctx.show_answer_spinner
    );
}
```

... and then just reference the `showAnswerButton` variable here.